v20.1/show-roles.md

```diff
@@ -18,7 +18,7 @@ The `SHOW ROLES` [statement](sql-statements.html) lists the roles for all databa
 
 ## Required privileges
 
-The user must have the [`SELECT`](select-clause.html) [privilege](authorization.html#assign-privileges) on the system table.
+The role must have the [`SELECT`](select-clause.html) [privilege](authorization.html#assign-privileges) on the system table.
```

Comment:
This one you actually need select on system.users table

Reply:
Ditto for system.roles - I believe `system` should be replaced with `system.users`